### PR TITLE
Adding user contribution child forms

### PR DESCRIPTION
**Pull request by RaimondSob**

### CODE REVIEW

File: Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+    public partial class UserContribution : Form
+    {
+        public UserContribution(Drink drink, Bar bar)
+        {
+            InitializeComponent();
+        }
+
+        
+        private void buttonAddDrink_Click(object sender, System.EventArgs e)
```
Comment (Raeppa) [inline Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs:21]: Capital letters for methods

File: Comparison Engine/Comparison Engine/Form1.cs [hunk 2]
```diff
@@ -136,7 +137,13 @@ private void openChildFormDrink(Drink drink)
             //closeActiveForm();            //#commentedarea
             configureChildForm(drinkForm);
         }
-        private void configureChildForm(Form childForm)
+        private void openChildFormUserContribution(Drink drink, Bar bar)
+        {
+            Child_Forms.UserContribution userContribution = new UserContribution(drink, bar);
```
Comment (Raeppa) [inline Comparison Engine/Comparison Engine/Form1.cs:142]: "Child_Forms." shouldn't be needed, since you already have the namespace imported

Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Form1.cs:142]: Did you check in with adam about this?

File: Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        {
+        }
+        private void buttonAddBar_Click(object sender, System.EventArgs e)
+        {
+            string BarName = textBoxBarName.Text;
+            string BarAdress = textBoxBarAdress.Text;
+            //AddBar(BarName, BarAdress);
+
+        }
```
Comment (pskramtai) [inline Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs:30]: Should use BarManager class to add bars and drinks. Also some ID generation should probably appear somewhere in this class

File: Comparison Engine/Comparison Engine/Form1.cs [hunk 4]
```diff
@@ -281,35 +288,38 @@ private void buttonTest3_Click(object sender, EventArgs e)
             }
         }
 
+        private void buttonEdit_Click(object sender, EventArgs e)
+        {
+            openChildFormUserContribution(drinks[2], bars[2]);
```
Comment (Raeppa) [inline Comparison Engine/Comparison Engine/Form1.cs:293]: why is the code referring to specifically the 3rd element here?

**Review by ElektrosStulpas — COMMENTED** [review]
hard to comment as not much code as of now

**Comment by ElektrosStulpas** [comment]
One more thing, need to capitalize methods

File: Comparison Engine/Comparison Engine/Form1.cs (hunk 2 above)
Comment (RaimondSob) [reply Comparison Engine/Comparison Engine/Form1.cs:142]: don't know why it shows that I deleted this, it still there in form1.cs

**Review by Raeppa — COMMENTED** [review]
Methods need capitalization and need clarification regarding that [3] call

File: Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs (hunk 3 above)
Comment (Raeppa) [reply Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs:30]: ID's, for now, are generated when you create a bar using the BarManager.AddBar method

File: Comparison Engine/Comparison Engine/Form1.cs (hunk 4 above)
Comment (RaimondSob) [reply Comparison Engine/Comparison Engine/Form1.cs:293]: for no actual reason, was for testing and forgot to change it

**Pull request merged**